src/Actions/Communication/UpdateCommunication.php

```diff
@@ -41,8 +45,38 @@ public function performAction(): Model
         $communication->fill($this->data);
         $communication->save();
 
+        if (! is_null($communicatables)) {
+            $existing = array_filter($communicatables, fn (array $item) => ! is_null(data_get($item, 'id')));
+            $new = array_filter($communicatables, fn (array $item) => is_null(data_get($item, 'id')));
+
+            $communication->communicatables()
+                ->whereIntegerNotInRaw('id', array_column($existing, 'id'))
+                ->delete();
+
+            if ($new) {
+                $communication->communicatables()->createMany($new);
+            }
+
+            if ($existing) {
```

Comment:
Why do you want to allow updating the pivot table? I'm okay with creating/deleting, but not updating.